packages/babel-helper-create-class-features-plugin/src/misc.ts

```diff
@@ -86,15 +104,21 @@ export function injectInitialization(path, constructor, nodes, renamer?) {
   }
 }
 
-export function extractComputedKeys(ref, path, computedPaths, file) {
-  const declarations = [];
+export function extractComputedKeys(
+  ref: t.Identifier,
+  path: NodePath<t.Class>,
+  computedPaths: NodePath<t.ClassProperty | t.ClassMethod>[],
+  file,
+) {
+  const declarations: t.Statement[] = [];
   const state = {
     classBinding: path.node.id && path.scope.getBinding(path.node.id.name),
     file,
   };
   for (const computedPath of computedPaths) {
     const computedKey = computedPath.get("key");
     if (computedKey.isReferencedIdentifier()) {
+      // @ts-expect-error TODO: make isReferencedIdentifier return path is Identifier
```

Comment:
If you'll want to open a PR to do this, it will be a welcome improvement!

Reply:
I will do that!